modin/experimental/core/execution/native/implementations/omnisci_on_native/partitioning/partition_manager.py

```diff
@@ -178,7 +178,13 @@ def _get_unsupported_cols(cls, obj):
 
             try:
                 at = pyarrow.Table.from_pandas(obj)
-            except (pyarrow.lib.ArrowTypeError, pyarrow.lib.ArrowInvalid) as e:
+            except (
+                pyarrow.lib.ArrowTypeError,
+                pyarrow.lib.ArrowInvalid,
+                ValueError,
+            ) as e:
```

Comment:
nitpick: e->err?  1-letter variable names make grepping hard

Reply:
Probably, it makes sense to create a separate, refactoring, PR and rename all other similar variables, not just this one.

Reply:
https://github.com/modin-project/modin/pull/5027

Reply:
@AndreyPavlenko Please make this change in your pull request as well.

Reply:
Done.